src/olympia/reviewers/views.py

```diff
@@ -629,6 +639,15 @@ def queue_expired_info_requests(request):
                   qs=qs, SearchForm=None)
 
 
+@permission_or_tools_view_required(amo.permissions.ADDONS_REVIEW)
+def queue_needs_human_review(request):
+    admin_reviewer = is_admin_reviewer(request)
+    qs = Addon.objects.get_needs_human_review_queue(
+        admin_reviewer=admin_reviewer)
+    return _queue(request, NeedsHumanReviewTable, 'needs_human_review',
+                  qs=qs, SearchForm=None)
```

Comment:
No `SearchForm`?

Reply:
I never include it if it's not requested in the issue cause reviewer tools search is a pain to work with. We can add it later if needed.